command/job_periodic_force_test.go

```diff
@@ -0,0 +1,204 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/helper"
+	"github.com/hashicorp/nomad/nomad/structs"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/nomad/nomad/mock"
+	"github.com/hashicorp/nomad/testutil"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+func TestJobPeriodicForceCommand_Implements(t *testing.T) {
+	t.Parallel()
+	var _ cli.Command = &JobPeriodicForceCommand{}
+}
+
+func TestJobPeriodicForceCommand_Fails(t *testing.T) {
+	t.Parallel()
+	ui := new(cli.MockUi)
+	cmd := &JobPeriodicForceCommand{Meta: Meta{Ui: ui}}
+
+	// Fails on misuse
+	if code := cmd.Run([]string{"some", "bad", "args"}); code != 1 {
+		t.Fatalf("expected exit code 1, got: %d", code)
```

Comment:
I'd suggest using require here as well, like you've done in rest of file - the error message are better with testify assertions than with basic library.